### DISCUSSION TITLE
Misuratore potenza pannello fotovoltaico

### DISCUSSION THREAD

**Discussion by ghost** [indomus/forum#1562] [HUB: Home Assistant]
Ciao a tutti, ho acquistato un pannello fotovoltaico da balcone e vorrei ora installare un misuratore che mi consenta di sapere quanta potenza eroga il pannello e quanto assorbo invece dalla rete generale. Sapreste consigliarmi un misuratore WiFi compatibile con HA da montare sul contatore di casa? Su Internet avevo visto un prodotto di iammeter ma è per me fuori budget (vorrei spendere meno di 100 euro).
Grazie a chi potrà aiutarmi
Stefano

**Comment by indomus** [comment]
Ciao Stefano, valuta sensori come l'ottimo [Shelly EM](https://indomus.it/componenti/shelly-em/) o analoghi.
